homeassistant/components/sensor/zha.py

```diff
@@ -143,3 +146,20 @@ def state(self):
             return None
 
         return round(float(self._state))
+
+
+class IlluminanceMeasurementSensor(Sensor):
+    """ZHA lux sensor."""
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement of this entity."""
+        return 'lx'
+
+    @property
+    def state(self):
+        """Return the state of the entity."""
+        if self._state is None:
```

Comment:
The isn't needed. Just return `self._state`

Reply:
Was just following what other sensors were doing already. I can change this though. I’ll get it updated soon.

Reply:
If the state is already `None` this condition is useless.